docs/Example Schedules/CE Schedules/index.md

```diff
@@ -1 +1,3 @@
-# Coming soon!
\ No newline at end of file
+# Coming soon!
```

Comment:
Not completely sure what to do with this page right now, so just leave it be and I'll update it later